kahuna/public/js/image/controller.js

```diff
@@ -170,6 +176,17 @@ image.controller('ImageCtrl', [
                 });
         };
 
+        ctrl.delete = function() {
+            const confirmed = window.confirm('Are you sure you want to delete this image from the Grid?');
```

Comment:
We have confirm delete pattern [here](https://github.com/guardian/grid/blob/master/kahuna/public/js/components/gr-confirm-delete/gr-confirm-delete.js) that matches the rest of the tools.